definition-map.js

```diff
@@ -73,9 +73,8 @@ const QUERIES = {
         args: {
             id: "The ID of the Topic.",
             topicType: "The type of the Topic.",
-            linkedWorkbook: "The ID of the linked workbook.",
```

Comment:
Based on the changes here:

https://github.com/thoughtindustries/ti/pull/8759/files#diff-bfe9efc9b7b746e4a99142ba5cbd729f4e6132e00788ce83327bacf84457ac23R96

We should still have a `linkedWorkbookId` property, right?

Reply:
Since `linkedWorkbookId` is detected as an id type field, it is now handled by `augment-schema.js`. Thought we could remove the manual definition from `definition-map.js` here? verified in the updated schema and the documentation.

Reply:
@ljiang-ti Yeah you're right - I was just thrown off by the inclusion of `instructorAssessmentUserId` [here](https://github.com/thoughtindustries/helium-graphql/pull/18/files#diff-81f78d4936409cba2406ac8bdcd37bcd5b190a97e4d6040f389e4add5e5b2e48R77).

Re-reading the description though, I'm guessing it's because the id type field handling in `augment-schema.js` wouldn't generate a particularly helpful description, yeah?

Reply:
Yup, thought leaving the custom description of `instructorAssessmentUserId` would be helpful.